arviz/rcparams.py

```diff
@@ -262,6 +262,7 @@ def validate_iterable(value):
         "mean",
         _make_validate_choice({"mean", "median", "mode"}, allow_none=True),
     ),
+    "plot.band_prob": (0.95, _validate_probability),
```

Comment:
Potentially this should be `stats.band_prob`, especially if we add the ECDF confidence band computation functions to the API.